RecoLocalMuon/GEMRecHit/plugins/GEMRecHitProducer.cc

```diff
@@ -81,6 +83,7 @@ void GEMRecHitProducer::fillDescriptions(edm::ConfigurationDescriptions& descrip
   desc.add<std::string>("recAlgo", "GEMRecHitStandardAlgo");
   desc.add<edm::InputTag>("gemDigiLabel", edm::InputTag("muonGEMDigis"));
   desc.add<bool>("applyMasking", false);
+  desc.add<bool>("ge21Off", false);
```

Comment:
@watson-ij 

I'd like to ask a clarification for HLT. This PR (and backports) is updating the offline reconstruction, but not the HLT menu(s). This update via `fillDescription` will lead to `hltGemRecHits[GEMRecHitProducer].ge21Off = False` (default value) at HLT, in combination with `hltMuonGEMDigis[GEMRawToDigiModule].ge21Off = True` (see https://github.com/cms-sw/cmssw/issues/37556). You can check the current HLT menu [here](https://raw.githubusercontent.com/cms-sw/cmssw/2c4530492059ae8af645307b4f52d1df881b2ccc/HLTrigger/Configuration/test/OnLine_HLT_GRun.py).

Could you please expand on what are the correct settings for HLT in light of this PR? (cc: @Martin-Grunewald)

Reply:
to attn of @jshlee

Reply:
@missirol We are turning on the digi unpacking in offline so we can monitor the GE2/1 demonstrator, and turning off digi->rechit reconstruction so that the demonstrator still doesn't affect the muons. Since the GE2/1 digi unpacking is off at HLT (and I don't think theres a reason to turn it on), I don't think its necessary to change anything. I guess for safety, one could also set the ge21off in rechitproducer to True, but since there are no GE2/1 digis it should be a no-op.

Reply:
@watson-ij 

Okay. Thanks for clarifying!